src/assets/Ast.js

```diff
@@ -0,0 +1,18 @@
+class CSSAst {
```

Comment:
Rename the file to CSSAst, as js and html also have an ast...

Reply:
I was thinking that it would be better for future to have common source of asts?

Reply:
In the future every asset will have it's own package so it will be seperate anyways